test/NuGet.Core.Tests/NuGet.Build.Tasks.Pack.Test/PackTaskLogicTests.cs

```diff
@@ -79,6 +79,25 @@ public void PackTaskLogic_ErrorsOnBadFrameworkPlatform()
             }
         }
 
+        [Fact]
+        public void PackTaskLogic_WarnsMissingDot()
+        {
+            // Arrange
+            using (var testDir = TestDirectory.Create())
+            {
+                var tc = new TestContext(testDir, "net50");
+
+                // Act
+                tc.BuildPackage();
+
+                // Assert
+                var logger = (TestLogger)tc.Request.Logger;
+                var messages = logger.WarningMessages.ToArray();
+                Assert.True(messages[0].Contains("net50"));
+                Assert.True(messages[0].Contains("missing dot"));
+            }
+        }
+
```

Comment:
It is better if we add another unit test to test `tfms` with `-` char in it. we can pick up an example from [net5 design doc](https://github.com/terrajobst/designs/blob/master/accepted/2020/net5/net5.md#os-versions). Please let me know if I am missing anything here.

Reply:
haha good call on this. Thank you! There was a bug in my logic 👍🏻